x/evm/keeper/abci.go

```diff
@@ -27,3 +34,49 @@ func (k *Keeper) EndBlock(ctx sdk.Context, _ abci.RequestEndBlock) []abci.Valida
 
 	return []abci.ValidatorUpdate{}
 }
+
+// updateBlockBloom checks for eth_tx_log events at BeginBlock
+// only those on the modules BeginBlocker should be included (eg. epochs)
+// and update block bloom filter with these
+func (k *Keeper) updateBlockBloom(ctx sdk.Context) {
+	logger := ctx.Logger().With("begin_block", "evm")
```

Comment:
Any particular reason here we are using the ctx logger instead of the keeper one?

Reply:
Yes the default one has arguments `module` and `evm` so this is more precise where the log is happening.